### ISSUE TITLE
Request SUPER+k hotkey interceptor 

### ISSUE THREAD

**Issue by joshmanders** [soloterm/solo#32]
Not entirely sure how this would work and what the implications of it are when ran inside things such as VSCode's terminal or what, but on macOS `cmd+k` in the terminal clears the terminal as if you typed `clear` manually. I use this extensively to a point that it's my muscle memory to hit it when I'm inside a solo tab such as logs.

While solo redraws the UI every tick, it's not a gigantic issue because if I clear out the screen like that and solo "disappears" I just hit one of the other usual hotkeys such as left or right to cause the screen the redraw. But it would be nice to intercept `cmd+k` and map it to the clear command inside solo.

**Comment by aarondfrancis** [comment]
Yes! Love this. I have a WIP PR #4 that I accidentally did one million things on, but that includes the ability to register custom hotkeys / palettes. For example, here's the WIP of a Vim palette: https://github.com/aarondfrancis/solo/pull/4/files#diff-72e45d27c61e4656602b1f5c3902c6e848fb131919bfd24e207f62918989b7e5. 

So you could register a Josh set of keys and include cmd k in that. I'll try to get that merged soon!

**Comment by joshmanders** [comment]
I <3 you, Aaron.

**Comment by aarondfrancis** [comment]
Ok hotkeys are in, you can register whatever you want! Lemme know if that fixes it
